src/platform-includes/performance/automatic-instrumentation-intro/javascript.mdx

```diff
@@ -1 +1 @@
-The `@sentry/tracing` package provides a `BrowserTracing` integration to add automatic instrumentation for monitoring the performance of browser applications.
+The `@sentry/browser` package provides a `BrowserTracing` integration to add automatic instrumentation for monitoring the performance of browser applications.
```

Comment:
l: I'd keep this more general because this include is shown for every SDK package:
 
```suggestion
The Sentry SDK provides a `BrowserTracing` integration to add automatic instrumentation for monitoring the performance of browser applications.
```